README.md

```diff
@@ -1,2 +1,38 @@
-# gsoc_relationship
-WIP: this repository is for organizing the gsoc relationship project
+# GSoC: Holmes Automated Malware Relationships (WIP)
+
+## Introduction
+
+![GitHub Logo](/images/architecture_1.png)
```

Comment:
Might I suggest you change the title of "Relationship Score Generator" to "Final Relationship Generator." While scoring is a huge part, it should also be sorting and merging the primary relationships into sets of final relationships. 

Additionally, I would add a box for "raw output" under d3.js. d3.js is a huge component and the primary goal. However, outputting the raw results of the final relationships should be captured as well.